site/index.ejs

```diff
@@ -400,10 +515,10 @@ description: A tool to help you track down COVID-19 vaccine appointment openings
       <div class="container">
         <h3>Tool Status</h3>
         <ul>
-          <li><strong>CVS:</strong> Last checked <%= (new Date(this.lodash.orderBy(cvs, ['lastFetched'], ['desc'])[0].lastFetched)).toLocaleDateString('en-US', { dateStyle: 'long', timeStyle: 'long', timeZone: 'America/Denver' }) %>. Checking <%= cvs.length %> <%= (cvs.length === 1) ? 'city' : 'cities' %> (new cities will be added as CVS adds them).</li>
-          <li><strong>Safeway, Albertsons:</strong> Last checked <%= (new Date(this.lodash.orderBy(albertsons, ['lastFetched'], ['desc'])[0].lastFetched)).toLocaleDateString('en-US', { dateStyle: 'long', timeStyle: 'long', timeZone: 'America/Denver' }) %>. Checking all <%= albertsons.length %> <%= (albertsons.length === 1) ? 'store' : 'stores' %> in CO.</li>
-          <li><strong>Walgreens:</strong> Last checked <%= (new Date(this.lodash.orderBy(walgreens, ['lastFetched'], ['desc'])[0].lastFetched)).toLocaleDateString('en-US', { dateStyle: 'long', timeStyle: 'long', timeZone: 'America/Denver' }) %>. Checking all <%= walgreens.length %> <%= (walgreens.length === 1) ? 'store' : 'stores' %> in CO.</li>
-          <li><strong>Walmart:</strong> Last checked <%= (new Date(this.lodash.orderBy(walmart, ['lastFetched'], ['desc'])[0].lastFetched)).toLocaleDateString('en-US', { dateStyle: 'long', timeStyle: 'long', timeZone: 'America/Denver' }) %>. Checking all <%= walmart.length %> <%= (walgreens.length === 1) ? 'store' : 'stores' %> in CO.</li>
+          <li><strong>CVS:</strong> Last checked <%= //(new Date(this.lodash.orderBy(cvs, ['lastFetched'], ['desc'])[0].lastFetched)).toLocaleDateString('en-US', { dateStyle: 'long', timeStyle: 'long', timeZone: 'America/Denver' }) %>. Checking <%= cvs.length %> <%= (cvs.length === 1) ? 'city' : 'cities' %> (new cities will be added as CVS adds them).</li>
```

Comment:
These are simply commented out for now because in my case where I have no Albertson's stores fetched and do not have the Walmart scraper going, they cause template rendering to fail. 

I will look into making this more robust to handle this case.